apps/libsignpost/sara_u260.h

```diff
@@ -9,6 +9,16 @@
 #define SARA_U260_INVALIDPARAM -2
 #define SARA_U260_NO_SERVICE -3
 
+typedef struct __attribute__((packed)) sara_u260_ops_info {
+    uint16_t mcc;
+    uint16_t mnc;
+    uint16_t lac;
+    uint32_t ci;
+    uint8_t  bsic;
+    uint16_t arfcn;
+    uint8_t  rxlev;
+} sara_u260_ops_info_t;
+
```

Comment:
I agree with leaving the variable names to match the spec, but could you add comments for what all these acronyms are?